phpcs.xml.dist

```diff
@@ -125,6 +125,7 @@
   <rule
     ref="SlevomatCodingStandard.TypeHints.NullableTypeForNullDefaultValue"
   />
+  <rule ref="SlevomatCodingStandard.Functions.StaticClosure" />
```

Comment:
Nit: this is now in-between all the `TypeHints` rules.

Can we move this before or after `SlevomatCodingStandard.Functions.ArrowFunctionDeclaration` so the `Functions` rules are grouped/

Reply:
Good point. In https://github.com/GoogleForCreators/web-stories-wp/pull/12908/commits/31cb79823c6e75122d71b821e59f19e6feb9eb83 I sort the groups, so it easier to keep track of.